python/sidecar/projects/schemas.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import annotations
```

Comment:
@cguedes I slightly changed a few of the `projects` related return types, so let me know if you have any issues with this. When we were first writing this backend, we were just returning `str` and `dict` all over, so did my best to make proper types.

Reply:
I also pushed with `--no-verify` as the `projects` type changes resulted in some small issues on the TS side, but I can give fixing those a shot once we're aligned on the changes here.